idaes/generic_models/unit_models/plate_heat_exchanger.py

```diff
@@ -429,160 +485,200 @@ def rule_cp_cold(blk, t):
 
         # =====================================================================
         # PERFORMANCE EQUATIONS
-        # mass flow rate in kg/s
-        def rule_mh_in(blk, t):
-            return blk.hot_fluid.properties_in[t].flow_mol *\
-                blk.hot_fluid.properties_in[t].mw
-        self.mh_in = Expression(self.flowsheet().time, rule=rule_mh_in,
-                                doc='Hotside mass flow rate [kg/s]')
-
-        def rule_mc_in(blk, t):
-            return blk.cold_fluid.properties_in[t].flow_mol *\
-                blk.cold_fluid.properties_in[t].mw
-        self.mc_in = Expression(self.flowsheet().time, rule=rule_mc_in,
-                                doc='Coldside mass flow rate [kg/s]')
 
-        # ---------------------------------------------------------------------
-        # port mass velocity[kg/m2.s]
+        # G:mass flow velocity, p :port , h:hotfluid , c :coldfluid
+        # Gph is hotfluid port mass velocity[kg/m2.s]
         def rule_Gph(blk, t):
-            return (4 * blk.mh_in[t] * 7) / (22 * blk.port_dia**2)
+            return (4 * blk.hot_fluid.properties_in[t].flow_mass * 7) /\
```

Comment:
Again, I would like to know where these hard-coded constants are coming from. A comment may suffice.